source/exec/docker.d

```diff
@@ -135,8 +136,15 @@ class Docker: IExecProvider
 			sleep(300.msecs);
 		}
 
+		// remove coloring
+		auto removeColoring = pipeProcess(["sed", "-r", `s/\x1B\[([0-9]{1,2}(;[0-9]{1,2})?)?[mGK]//g`], Redirect.stdout | Redirect.stderrToStdout | Redirect.stdin);
```

Comment:
An alternative to this crude `sed` call, would be to build sth. like [`aha`](https://github.com/theZiz/aha).
This would be pretty sweat because we could then also for the normal builds, call `dmd` with `-color`, but as this drastically complicates the build process, I leave this up for another PR.